weldx/geometry.py

```diff
@@ -2089,3 +2088,36 @@ def rasterize(self, profile_raster_width, trace_raster_width):
         return self._rasterize_variable_profile(
             profile_raster_width, trace_raster_width
         )
+
+    def plot(self, profile_raster_width, trace_raster_width, axes=None, fmt=None):
+        """Plot the geometry.
+
+        Parameters
+        ----------
+        profile_raster_width: float, int
+            Target distance between the individual points of a profile
+        trace_raster_width: float, int
+            Target distance between the individual profiles on the trace
+        axes : matplotlib.axes.Axes
+            The target `Axes` object of the plot. If 'None' is passed, a new figure will
+            be created
+        fmt : str
+            Format string that is passed to matplotlib.pyplot.plot.
+
+        Returns
+        -------
+
+        """
+        data = self.rasterize(profile_raster_width, trace_raster_width)
+        if fmt is None:
+            fmt = "o"
+        if axes is None:
+            fig = plt.figure()
+            axes = fig.gca(projection="3d")
```

Comment:
```suggestion
            axes = fig.gca(projection="3d", proj_type="ortho")
```

always use `proj_type="ortho"` for all 3d plots